labels.json

```diff
@@ -1,135 +1,316 @@
     "aliases": [
-      "needs-rebase",
-      "rebase"
+      "resolution: works as expected",
+      "not-a-bug"
+    ]
+  },
+  {
+    "name": "keep open",
+    "description": "An Issue or Pull Request that should be kept open",
+    "color": "fbca04",
+    "aliases": [
+      "keep open",
+      "leave-open"
     ]
   },
   {
-    "name": "needs-squash",
+    "name": "for new contributors",
+    "description": "An Issue raising a need for changes, that should be relatively easy to do",
+    "color": "02d7e1",
+    "aliases": [
+      "for new contributors",
+      "beginner",
+      "beginner-friendly",
+      "beginner friendly"
+    ]
+  },
+  {
+    "name": "status: confirmed",
+    "description": "An Issue reporting a bug that is confirmed to affect other users",
     "color": "c2e0c6",
     "aliases": [
-      "needs-squash",
-      "squash"
+      "status: confirmed",
+      "confirmed"
     ]
   },
   {
-    "name": "needs-tests",
-    "color": "f7c6c7",
+    "name": "status: unconfirmed",
+    "description": "An Issue reporting a potential bug that is yet to be confirmed by other users",
+    "color": "fef2c0",
+    "aliases": [
+      "status: unconfirmed",
+      "unconfirmed"
+    ]
+  },
+  {
+    "name": "security",
+    "description": "Any Issue or Pull Request that directly relates to security",
+    "color": "ff1d21",
+    "aliases": [
+      "security"
+    ]
+  },
+  {
+    "name": "type: chore",
+    "description": "A Pull Request that is raised as part of routine module maintenance",
+    "color": "e7ed39",
+    "aliases": [
+      "type: chore",
+      "chore",
+      "modulesync"
+    ]
+  },
+  {
+    "name": "type: feature",
+    "description": "A Pull Request raised that adds completely new functionality",
+    "color": "c7def8",
+    "aliases": [
+      "type: feature",
+      "feature"
+    ]
+  },
+  {
+    "name": "type: enhancement",
+    "description": "A Pull Request raised that enhances existing functionality",
+    "color": "84b6eb",
+    "aliases": [
+      "type: enhancement",
+      "enhancement"
+    ]
+  },
+  {
+    "name": "type: refactor",
+    "description": "A Pull Request raised that involves a large number of changes to module design",
+    "color": "eb6420",
+    "aliases": [
+      "type: refactor",
+      "refactor"
+    ]
+  },
+  {
+    "name": "type: test",
+    "description": "A Pull Request raised that adds one or more tests",
+    "color": "bfe5Bf",
+    "aliases": [
+      "type: test",
+      "test"
+    ]
+  },
+  {
+    "name": "type: deprecation",
+    "description": "A Pull Request raised that deprecates existing functionality",
+    "color": "eb6420",
+    "aliases": [
+      "type: deprecation"
+    ]
+  },
+  {
+    "name": "needs: docs",
+    "description": "A Pull Request raised that needs associated documentation changes",
+    "color": "5319e7",
+    "aliases": [
+      "needs: docs",
+      "needs-docs"
+    ]
+  },
+  {
+    "name": "needs: feedback",
```

Comment:
A PR request that involves seems like a major change in direction and have long-lived consequences should be discussed a little.